lib/pages/Home.dart

```diff
@@ -158,127 +162,121 @@ class Home extends StatelessWidget {
                           delegate: SliverChildListDelegate(
-                        [
-                          Padding(
-                            padding: EdgeInsets.symmetric(horizontal: 16),
-                            child: Column(
-                                crossAxisAlignment: CrossAxisAlignment.start,
-                                children: [
+                            [
+                              Padding(
+                                padding: EdgeInsets.symmetric(horizontal: 16),
+                                child: Column(
+                                    crossAxisAlignment: CrossAxisAlignment.start,
+                                    children: [
+                                      Text(
+                                        'Current Ride',
+                                        style: CarriageTheme.subHeading,
+                                      ),
+                                      SizedBox(height: 12),
+                                      CurrentRideCard(ridesProvider.currentRide, showCallDriver: true),
+                                    ]),
+                              ),
+                              SizedBox(height: 35),
+                              Padding(
+                                padding: EdgeInsets.symmetric(horizontal: 16),
+                                child: Row(children: [
                                   Text(
-                                    'Current Ride',
+                                    'Upcoming Rides',
                                     style: CarriageTheme.subHeading,
                                   ),
-                                  SizedBox(height: 12),
-                                  CurrentRideCard(ridesProvider.currentRide,
-                                      showCallDriver: true),
-                                ]),
-                          ),
-                          SizedBox(height: 35),
-                          Padding(
-                            padding: EdgeInsets.symmetric(horizontal: 16),
-                            child: Row(children: [
-                              Text(
-                                'Upcoming Rides',
-                                style: CarriageTheme.subHeading,
-                              ),
-                              Spacer(),
-                              ridesProvider.upcomingRides.isNotEmpty
-                                  ? GestureDetector(
+                                  Spacer(),
+                                  ridesProvider.upcomingRides.isNotEmpty ? GestureDetector(
                                       onTap: () => Navigator.of(context).push(
-                                          MaterialPageRoute(
-                                              builder: (context) =>
-                                                  UpcomingSeeMore())),
+                                          MaterialPageRoute(builder: (context) => UpcomingSeeMore())
```

Comment:
On my end, when I go on UpcomingSeeMore, I am unable to see any rides. However, the upcoming rides are displayed properly on the home page

Reply:
Will be fixed in separate PR